### PR TITLE
show ServiceOfNotice details tab

### PR DESCRIPTION
**Pull request by stevelee-civica**
76683 ServiceOfNotice Details Tab

### CODE REVIEW

**Comment by seanciv** [comment]
Looks good, but do we need tests for ServiceOfNoticeDetails()? @stevelee-civica

**Comment by stevelee-civica** [comment]
added unit tests

File: src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs [hunk 1]
```diff
@@ -396,14 +406,48 @@ public async Task OrganisationDetails_SetViewBreadcrumb()
...
         }
+
+        [Fact]
+        public async Task ServiceOfNoticeDetails_Get_ReturnViewModelAndMapsData()
+        {
+            // Arrange
+            SetupDefaultControllerData();
+
+            var source = new SmallProducerSubmissionMapperData
+            {
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs:437]: @stevelee-civica  Think this test should test that the mapper is called and that the ServiceOfNoticeViewModel property on the model is that value returned from the A.CallTo(.....

File: src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs [hunk 2]
```diff
@@ -448,6 +448,10 @@ public async Task ServiceOfNoticeDetails_Get_ReturnViewModelAndMapsData()
 
             model.Should().NotBeNull();
             model.ServiceOfNoticeViewModel.Should().NotBeNull();
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs:445]: @stevelee-civica can you setup 
 A.CallTo(() => mapper
                            .Map<SmallProducerSubmissionMapperData, ServiceOfNoticeViewModel>(A<SmallProducerSubmissionMapperData>.That.Matches(sd => sd.SmallProducerSubmissionData.Equals(controller.SmallProducerSubmissionData))))
                            
                            
to return a ServiceOfNoticeViewModel and check that model.ServiceOfNoticeViewModel.Should().be(the expected value)

Comment (stevelee-civica) [reply src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs:460]: updated

**Pull request merged**